src/framework/preferences/css/stylus/PrefsEditor.styl

```diff
@@ -169,6 +169,10 @@
         .fl-preview-A {
             font-size: 2em;
         }
+        
+        .fl-preview-A:before {
+            content: "a";
```

Comment:
I wonder if the content should be moved back to the markup for the label. It would be easier to change this all in one place and for localization purposes as well. What do you think?